Mathlib/Order/CompleteLattice.lean

```diff
@@ -540,6 +540,12 @@ theorem sInf_eq_bot : sInf s = ⊥ ↔ ∀ b > ⊥, ∃ a ∈ s, a < b :=
   @sSup_eq_top αᵒᵈ _ _
 #align Inf_eq_bot sInf_eq_bot
 
+lemma sup_eq_top_of_top_mem (h : ⊤ ∈ s) : sSup s = ⊤ :=
```

Comment:
This should be true in a more general case. At least, for a complete lattice. Possibly, for a conditionally complete lattice with top.

Reply:
The proof for a conditionally complete lattice + `OrderTop` is `top_unique <| le_csSup (OrderTop.bddAbove s) h`

Reply:
You are right, thank you!